plugins/README.md

```diff
@@ -0,0 +1,51 @@
+# Oh My Bash: Enhancing Your Development Experience
```

Comment:
This README is the one for OMB plugins, so "Oh My Bash" doesn't seem the correct title. For example,

```suggestion
# Oh-My-Bash Plugins: Enhancing Your Development Experience
```